docker-local/docker/wait4/wait4.config.js

```diff
@@ -96,6 +96,36 @@ module.exports = {
           method: 'ncat'
         }
       ]
+    },
+    {
+      name: 'pisp-thirdparty-scheme-adapter',
+      wait4: [
+        {
+          description: 'PISP Redis Cache',
+          uri: 'pisp-redis:6379',
```

Comment:
do we want to use one Redis instance for all services?

Reply:
hmm. I don't have enough experience to answer.
for simplicity? maybe. I'm not sure.

Reply:
Yeah maybe we can, that would make this config a lot simpler

Reply:
Although it will make it harder for others to read and understand our `docker-compose` file...